lib/puppet/pops/parser/lexer2.rb

```diff
@@ -500,6 +500,7 @@ def initialize()
         else
           @scanner.pos += 1
         end
+        ctx[:current_line_start] = @scanner.pos
```

Comment:
Looks like this will be wrong after a newline_jump (line 498) as that position is not at the beginning of the line IIRC.